src/System.Threading.Tasks/src/System.Threading.Tasks.csproj

```diff
@@ -27,7 +28,6 @@
     <Compile Include="System\Threading\Tasks\TaskExtensions.CoreRT.cs" />
     <ProjectReference Include="..\..\System.Runtime\src\redist\System.Runtime.depproj" />
     <TargetingPackReference Include="System.Private.CoreLib" />
-    <TargetingPackReference Include="System.Private.CoreLib.Threading" />
```

Comment:
Doesn't this break? As I would expect it to still need the System.Private.CoreLib.Threading for the netcore50aot build configuration.